sklearn/utils/tests/test_shortest_path.py

```diff
@@ -9,21 +9,14 @@
 def floyd_warshall_slow(graph, directed=False):
```

Comment:
The original is really the pseudo-code of the Floyd Warshall algorithm: https://en.wikipedia.org/wiki/Floyd%E2%80%93Warshall_algorithm

```
let dist be a |V| × |V| array of minimum distances initialized to ∞ (infinity)
for each edge (u, v) do
    dist[u][v] ← w(u, v)  // The weight of the edge (u, v)
for each vertex v do
    dist[v][v] ← 0
for k from 1 to |V|
    for i from 1 to |V|
        for j from 1 to |V|
            if dist[i][j] > dist[i][k] + dist[k][j] 
                dist[i][j] ← dist[i][k] + dist[k][j]
            end if
```

If the outer product, this is much more difficult to make the link with the pseudo-code.

I would be -1 on this change.